File: frontend/src/services/UpdateEmployerInfoService.js
import axios from "axios"
import Links from "../components/Links"

export default class UpdateEmployerInfoService{
    get() {
        return axios.get(Links.ROOT + "/updateEmployerInfo/getAll")
    }

    add(userId, values) {
        return axios.post(`${Links.ROOT} /updateEmployerInfo/add?userId=` + userId, values)
    }

    getById(userId) {
        return axios.get(Links.ROOT + "/updateEmployerInfo/getByUserId?userId=" + userId)
    }

    confirmByAdmin(userId) {
        return axios.post(`${Links.ROOT} /updateEmployerInfo/confirmByAdmin?userId=` + userId)
    }

}

File: backend/src/main/java/kodlamaio/hrms/business/abstracts/UpdateEmployerInfoService.java
package kodlamaio.hrms.business.abstracts;

import kodlamaio.hrms.core.utilities.results.DataResult;
import kodlamaio.hrms.core.utilities.results.Result;
import kodlamaio.hrms.entities.concretes.UpdateEmployerInfo;
import kodlamaio.hrms.entities.dtos.input.EmployerInputDto;

public interface UpdateEmployerInfoService extends BaseService<UpdateEmployerInfo>{
	Result add(int userId, EmployerInputDto entity);
	Result confirmByAdmin(int id);
	DataResult<UpdateEmployerInfo> getByUserId(int userId);
}
